diff --git a/app/src/lib/stores/app-store.ts b/app/src/lib/stores/app-store.ts
index e8b8de9..cc962f0 100644
--- a/app/src/lib/stores/app-store.ts
+++ b/app/src/lib/stores/app-store.ts
@@ -3357,6 +3357,8 @@ export class AppStore extends TypedBaseStore<IAppState> {
 
     if (selectedSection === RepositorySectionTab.History) {
       await this.refreshHistorySection(repository)
+    } else if (selectedSection === RepositorySectionTab.Commits) {
+      await this.refreshHistorySection(repository)
     } else if (selectedSection === RepositorySectionTab.Changes) {
       await this.refreshChangesSection(repository, {
         includingStatus: true,
@@ -4107,6 +4109,8 @@ export class AppStore extends TypedBaseStore<IAppState> {
 
     if (section === RepositorySectionTab.History) {
       refreshSectionPromise = this.refreshHistorySection(repository)
+    } else if (section === RepositorySectionTab.Commits) {
+      refreshSectionPromise = this.refreshHistorySection(repository)
     } else if (section === RepositorySectionTab.Changes) {
       refreshSectionPromise = this.refreshChangesSection(repository, {
         includingStatus: false,
