diff --git a/app/src/lib/menu-update.ts b/app/src/lib/menu-update.ts
index 373ffae..9388da2 100644
--- a/app/src/lib/menu-update.ts
+++ b/app/src/lib/menu-update.ts
@@ -126,6 +126,7 @@ const allMenuIds: ReadonlyArray<MenuIDs> = [
   'create-branch',
   'show-changes',
   'show-history',
+  'show-commits',
   'show-repository-list',
   'show-branches-list',
   'open-working-directory',
@@ -246,6 +247,7 @@ function getRepositoryMenuBuilder(state: IAppState): MenuStateBuilder {
     'go-to-commit-message',
     'show-changes',
     'show-history',
+    'show-commits',
     'show-branches-list',
     'show-worktrees-list',
     'open-external-editor',
diff --git a/app/src/main-process/menu/build-default-menu.ts b/app/src/main-process/menu/build-default-menu.ts
index cd5c20c..7b4a875 100644
--- a/app/src/main-process/menu/build-default-menu.ts
+++ b/app/src/main-process/menu/build-default-menu.ts
@@ -191,6 +191,12 @@ export function buildDefaultMenuTemplate({
         accelerator: 'CmdOrCtrl+2',
         click: emit('show-history'),
       },
+      {
+        label: __DARWIN__ ? 'Show Commits' : 'Show Co&mmits',
+        id: 'show-commits',
+        accelerator: 'CmdOrCtrl+3',
+        click: emit('show-commits'),
+      },
       {
         label: __DARWIN__ ? 'Show Repository List' : 'Repository &list',
         id: 'show-repository-list',
diff --git a/app/src/main-process/menu/menu-event.ts b/app/src/main-process/menu/menu-event.ts
index aa18eac..d0f8b7a 100644
--- a/app/src/main-process/menu/menu-event.ts
+++ b/app/src/main-process/menu/menu-event.ts
@@ -5,6 +5,7 @@ export type MenuEvent =
   | 'fetch'
   | 'show-changes'
   | 'show-history'
+  | 'show-commits'
   | 'add-local-repository'
   | 'create-branch'
   | 'show-branches'
diff --git a/app/src/models/menu-ids.ts b/app/src/models/menu-ids.ts
index 4365261..775aaa4 100644
--- a/app/src/models/menu-ids.ts
+++ b/app/src/models/menu-ids.ts
@@ -22,6 +22,7 @@ export type MenuIDs =
   | 'create-branch'
   | 'show-changes'
   | 'show-history'
+  | 'show-commits'
   | 'show-repository-list'
   | 'show-branches-list'
   | 'open-working-directory'
diff --git a/app/src/ui/app.tsx b/app/src/ui/app.tsx
index 568219f..a4ece00 100644
--- a/app/src/ui/app.tsx
+++ b/app/src/ui/app.tsx
@@ -465,6 +465,8 @@ export class App extends React.Component<IAppProps, IAppState> {
         return this.showChanges(true)
       case 'show-history':
         return this.showHistory(true)
+      case 'show-commits':
+        return this.showCommits(true)
       case 'choose-repository':
         return this.chooseRepository()
       case 'add-local-repository':
@@ -916,6 +918,28 @@ export class App extends React.Component<IAppProps, IAppState> {
     }
   }
 
+  private async showCommits(shouldFocusCommitList: boolean) {
+    const state = this.state.selectedState
+    if (state == null || state.type !== SelectionType.Repository) {
+      return
+    }
+
+    await this.props.dispatcher.closeCurrentFoldout()
+
+    await this.props.dispatcher.initializeCompare(state.repository, {
+      kind: HistoryTabMode.History,
+    })
+
+    await this.props.dispatcher.changeRepositorySection(
+      state.repository,
+      RepositorySectionTab.Commits
+    )
+
+    if (shouldFocusCommitList) {
+      this.repositoryViewRef.current?.setFocusCommitsNeeded()
+    }
+  }
+
   private async showChanges(shouldFocusChanges: boolean) {
     const state = this.state.selectedState
     if (state == null || state.type !== SelectionType.Repository) {
